lib/internal/webstreams/util.js

```diff
@@ -211,11 +211,12 @@ function createAsyncFromSyncIterator(syncIteratorRecord) {
   return { iterator: asyncIterator, nextMethod, done: false };
 }
 
+// Refs: https://tc39.es/ecma262/#sec-getiterator
 function getIterator(obj, kind = 'sync', method) {
   if (method === undefined) {
     if (kind === 'async') {
       method = obj[SymbolAsyncIterator];
-      if (method === undefined) {
+      if (method === undefined || method === null) {
```

Comment:
```suggestion
      if (method == null) {
```

`==`, IIRC, matches both `undefined` and `null`